src/HealthGPS/dynamic_hierarchical_linear_model.h

```diff
@@ -74,21 +73,21 @@ class EnergyBalanceHierarchicalModel final : public HierarchicalLinearModel {
 };
 
 /// @brief Defines the lite hierarchical linear model data type
```

Comment:
Maybe this docstrings needs updating, too?

Reply:
Ah, good spot, thanks!